pages/dao/[symbol]/proposal/components/instructions/raydium/AddLiquidity.tsx

```diff
@@ -192,16 +170,15 @@ const AddLiquidityRaydium = ({
   }, [form])
 
   const schema = yup.object().shape({
-    tokenAName: yup.string().required('Token A Name is required'),
-    tokenBName: yup.string().required('Token B Name is required'),
-    tokenAAmountIn: yup
+    liquidityPool: yup.string().required('Liquidity Pool is required'),
+    baseAmountIn: yup
       .number()
-      .moreThan(0, 'Amount for A token should be more than 0')
-      .required('Amount for A token is required'),
-    tokenBAmountIn: yup
+      .moreThan(0, 'Amount for Base token should be more than 0')
+      .required('Amount for Base token is required'),
+    quoteAmountIn: yup
       .number()
-      .moreThan(0, 'Amount for B token should be more than 0')
-      .required('Amount for B token is required'),
+      .moreThan(0, 'Amount for Quote token should be more than 0')
+      .required('Amount for Quote token is required'),
     governedAccount: yup
```

Comment:
Order here should be ideally the same as the one bellow (< GovernedAccountSelect> ....)